### PR TITLE
Replace lodash chain with native array chain

### PR DESCRIPTION
**Pull request by vietduc01100001**
Addresses: #7747

### CODE REVIEW

File: lib/plugins/aws/metrics/awsMetrics.js [hunk 1]
```diff
@@ -117,12 +117,11 @@ class AwsMetrics {
...
     if (metrics && metrics.length > 0) {
       const getDatapointsByLabel = Label =>
-        _.chain(metrics)
-          .flatten()
-          .filter({ Label })
-          .map('Datapoints')
-          .flatten()
-          .value();
+        metrics
+          .flat()
```
Comment (medikoo) [inline lib/plugins/aws/metrics/awsMetrics.js:121]: We need to maintain support for Node.js v6, so unfortunally we cannot rely on native `flat` (in that case we need to stick to `_.flatten`)

Comment (vietduc01100001) [reply lib/plugins/aws/metrics/awsMetrics.js:121]: I'll revert that.

Comment (vietduc01100001) [reply lib/plugins/aws/metrics/awsMetrics.js:121]: If I keep the `_.flatten` method, there will be 2 nested `_.flatten` call? Any suggestion?

```javascript
_.flatten(
  _.flatten(metrics)
    .filter(metric => metric.Label === Label)
    .map(metric => metric.Datapoints)
);
```

Comment (medikoo) [reply lib/plugins/aws/metrics/awsMetrics.js:121]: I think that's ok, as in original, we need to _flat_ twice

**Pull request merged**